ui/app/services/path-help.js

```diff
@@ -23,6 +23,7 @@ export function sanitizePath(path) {
 
 export default Service.extend({
   attrs: null,
+  dynamicApiPath: '',
```

Comment:
We don't need this set here if it's just being passed into `urlForItem` right?

Reply:
When writing a test, I noticed that it needs to be set as an empty string because it will hit getDynamicApiPath before result.apiPath is set.  It then hits it again when it is set.